app/code/BelodubrovskyiAn/MyCustomModule/Block/CustomBlock.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace BelodubrovskyiAn\MyCustomModule\Block;
+class CustomBlock extends \Magento\Framework\View\Element\Template
+{
+    const LESSON3_TEMPLATE = "BelodubrovskyiAn_MyCustomModule::lesson/myhomework.phtml";
+    /**
+     * add custom template
+     */
+    protected function _prepareLayout()
+    {
+        parent::_prepareLayout();
+        $this->setTemplate(self::LESSON3_TEMPLATE);
+        return $this;
+    }
+    public function getUrlToJsonResponse() {
```

Comment:
Please add new lines between the methods. Move the open curly brace to the next line, Please use phpdocs (phpstorm can generate it)

Reply:
fix

Reply:
fix